tests/test_daemon.cpp

```diff
@@ -96,6 +102,94 @@ struct StubNameGenerator : public mp::NameGenerator
     }
     std::string name;
 };
+
+class TestCreate final : public mp::cmd::Command
+{
+public:
+    using Command::Command;
+    mp::ReturnCode run(mp::ArgParser* parser) override
+    {
+        auto on_success = [](mp::CreateReply& /*reply*/) { return mp::ReturnCode::Ok; };
+        auto on_failure = [this](grpc::Status& status)
+        {
+            mp::CreateError create_error;
+            create_error.ParseFromString(status.error_details());
+            const auto errors = create_error.error_codes();
+
+            cerr << "fail: ";
+            if(errors.size() == 1)
```

Comment:
Clang-format inserts space between `if` and `{` for me. And for below few lines. Did you run it?

Reply:
Must have forgotten, sorry

Reply:
OK, I just pushed the fix. Man I need to write myself a clang-format hook...